config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml

```diff
@@ -383,7 +383,7 @@ periodics:
       - --cluster=bootstrap-e2e-gce-mz
       - --env=MULTIZONE=true
       # Where the nodes reside.  Master is in the first one.
-      - --env=E2E_ZONES=us-central1-a us-central1-b us-central1-f
+      - --env=E2E_ZONES=us-central1-a us-central1-b us-west1-b
```

Comment:
all of these zones probably need to change

Reply:
Yeah, I don't think we can stretch across regions as these will be different subnetwork objects

Reply:
it's probably fine given only central1-f is affected?